web/src/components/nav/betaNotice.js

```diff
@@ -0,0 +1,20 @@
+import { Route } from "react-router-dom";
+import { DismissableMessage } from "../ui";
+
+const MESSAGESTYLE = {
+	background: "#fec61e",
+	marginTop: "1em",
+	display: "inline-block",
+	width: "100%",
+};
+
+const MESSAGECONTENT =
+	"WOAQ is currently in early release. You may encounter bugs and data may not be displayed accurately.";
+
+export const BetaNotice = () => {
+	return (
+		<Route path={["/maps", "/upload"]}>
+			<DismissableMessage style={MESSAGESTYLE} content={MESSAGECONTENT} />
+		</Route>
+	);
```

Comment:
I'm still unsure if this should be its own separate component or if it should live within one of the router files for encapsulation. I'll let @TangoYankee make a call on this.

Reply:
I'm not sure what @wendy-wm-wu has in mind. It could be a great alternative. 

As I'm reading this implementation, I. Love. Small. Components. More generally, I like functions that only do one thing. They're often easier to understand, reuse, and debug. In this vein, I like how `BetaNotice` builds on `Route` and `DismissableMessage` before it is placed in `BrowserRouter`.